integration-tests/features/steps/stack_analysis.py

```diff
@@ -39,6 +39,7 @@ def wait_for_stack_analysis_completion(context, version=3, token="without"):
     200 OK : analysis is already finished
```

Comment:
i think it might be good if we create a new file `stack_analysis_V2.py `and have these test written , also separate files would not lead to any confusion between the functions of V1 and V2, and we can easily differentiate between them. just a suggestion happy with modifications though